.github/workflows/pull-request.yml

```
@@ -0,0 +1,265 @@
+          node-version: '22.x'
+
+      - name: Install Dependencies
+        run: npm install
+      
+      - name: Get changed TypeScript files
+        id: changed-files
+        uses: tj-actions/changed-files@v40
+          
+      - name: Run tests
+        if: steps.changed-files.outputs.only_changed != 'true'
+        run: npm run test -- --watchAll=false --coverage       
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)